app/pages/project/components/ProjectNavbar/helpers/userHasLabAccess.js

```diff
@@ -0,0 +1,11 @@
+function userHasLabAccess({ projectRoles, user }) {
+  return projectRoles.some((role) => {
+    if (role.links.owner.id === user.id) {
```

Comment:
[minor] Any chance that `role.links.owner` or `role.roles` would be null/undefined for any reason?

Reply:
If they are, then the API would be pretty broken. I could still add existential checks just to be sure.